core/src/main/java/com/linecorp/armeria/common/thrift/ThriftCallbackCompletableFuture.java

```diff
@@ -0,0 +1,22 @@
+package com.linecorp.armeria.common.thrift;
```

Comment:
I thought it is better to move these classes to some sort of util package. Anyways, I left this class as it is.